pages/api/topics.js

```diff
@@ -1,5 +1,27 @@
 import { topics } from "../api/data";
+//  Fetch calls to AEM
+
+//const { NEXTJS_ADOBE_API_KEY, NEXTJS_ADOBE_URL } = process.env
+const NEXTJS_ADOBE_URL = process.env.NEXTJS_ADOBE_URL
+  ? process.env.NEXTJS_ADOBE_URL
+  : "https://www.canada.ca/api/assets/decd-endc/content-fragments.json";
+
+//
+// Fetch ALL topics from Adobe
+//
```

Comment:
So, since our content is live on Canada.ca, it might be better to hide this value. Potentially better than having it as `true` above in the .env file is that we could make the environment variable equal to this URL, and then if true we just call the URL directly. That way, we would still be able to use our "if" check, but also hide it from the codebase.

Reply:
Maybe we just name the environment variable something like `CONTENT_API_URL` so it's not specific to Abode?

Reply:
done